jenkins/Jenkinsfile

```
@@ -0,0 +1,148 @@
+pipeline {
+    agent {label 'podman'}
+
+    parameters {
+        booleanParam(name: 'PUSH_IMAGE_TO_ECR', defaultValue: false, description: 'Push image to ECR?')
+    }
+
+    options {
+
+        disableConcurrentBuilds()
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '7', artifactNumToKeepStr: '7',
+                                  daysToKeepStr: '7', numToKeepStr: '7'))
+        timeout(time: 48, unit: 'HOURS')
+    }
+    environment {
+        CR_ALL = 'docker-all-nexus.meteoswiss.ch'
+        SPACK_TAG='dev_PR-12'
+
+        TAG = sh(
+            script: "echo \$(date +%g%m).${GIT_COMMIT}",
+            returnStdout: true
+        ).trim()
+
+        VERSION = sh(
+            returnStdout: true, 
+            script: 'if [ "$BRANCH_NAME" = "main" ]; then echo -n latest; elif [ -n "${TAG_NAME+1}" ]; then echo -n ${TAG_NAME}; else echo -n ${BRANCH_NAME}; fi'
+            )
+        PUBLIC_RELEASE = sh(
+            returnStdout: true, 
+            script: 'if [ "$BRANCH_NAME" = "main" ]; then echo -n 1; elif [ -n "${TAG_NAME+1}" ]; then echo -n 1; else echo -n 0; fi'
+            )
+
+        NAMESPACE = "numericalweatherpredictions/dispersionmodelling/flexpart-ifs/flexpart-containerize"
```

```suggestion
        NAMESPACE = "numericalweatherpredictions/dispersionmodelling/flexpart-ifs/flexpart"
```
The name flexpart is already descriptive enough, or is there already an image for that? 
OR could add ewc to the $TAG if not

---

No, there isn’t another flexpart-ifs/flexpart image, so I switched to that name to maintain consistency with the flexpart-cosmo naming convention.